cmd/gossamer/genesis.go

```diff
@@ -15,25 +15,25 @@ import (
 )
 
 func loadGenesis(ctx *cli.Context) error {
-	fig, err := getConfig(ctx)
+	currentConfig, err := getConfig(ctx)
 	if err != nil {
 		return err
 	}
 
 	// read genesis file
-	fp := getGenesisPath(ctx)
-	dataDir := fig.Global.DataDir
+	genesisPath := getGenesisPath(ctx)
+	dataDir := expandTildeOrDot(currentConfig.Global.DataDir)
 	if ctx.String(utils.DataDirFlag.Name) != "" {
-		dataDir = ctx.String(utils.DataDirFlag.Name)
+		dataDir = expandTildeOrDot(ctx.String(utils.DataDirFlag.Name))
 	}
-	log.Debug("Loading genesis", "genesisfile", fp, "datadir", dataDir)
+	log.Debug("Loading genesis", "genesisPath", genesisPath, "dataDir", dataDir)
 
-	gen, err := genesis.LoadGenesisData(fp)
+	gen, err := genesis.LoadGenesisData(genesisPath)
 	if err != nil {
 		return err
 	}
 
-	log.Info("🕸\t Initializing node", "name", gen.Name, "id", gen.Id, "protocolID", gen.ProtocolId, "bootnodes", common.BytesToStringArray(gen.Bootnodes))
+	log.Info("🕸\t Initializing node", "Name", gen.Name, "ID", gen.ID, "protocolID", gen.ProtocolID, "Bootnodes", common.BytesToStringArray(gen.Bootnodes))
```

Comment:
I am not opposed to capitalizing values but this is inconsistent with all other logs and `protocolID` is the only one not capitalized. Maybe keep the key values lowercase for now and we can look at log formatting in an issue on its own to make sure we are looking at how we are logging as a whole.

Reply:
Well the basic rule is you log the field name with the exact same name as the variable. protocolID is not missing, I am not sure what you mean.

Reply:
`"protocolID"` should be `"ProtocolID"` if you are following that rule

Reply:
oh you mean the log string, sure, good point :)